.changeset/small-bottles-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Fixed `ErrorPage` to correctly lazy import from `@itwin/itwinui-illustrations-react`.
```

Comment:
this one is interesting — we tested it in #1400, but it broke sometime after that (presumably because of #1452).

bundlers generally prefer simple/basic uses of dynamic imports, i.e. the actual string being passed to `import()`, without any variables or wrapper functions, so it is fixed now.

Reply:
To confirm, should we have/reintroduce tests for deprecated components like `ErrorPage` to catch regresses like these?

Reply:
we do have tests, but since we were previously using hacks for jest (which didn't support dynamic `import()`) we couldn't catch them